projects/istio/project.yaml

```diff
@@ -0,0 +1,9 @@
+homepage: "https://github.com/istio/istio"
+primary_contact: ""
+auto_ccs :
+  - "adam@adalogics.com"
```

Comment:
would you mind putting  istio-security-vulnerability-reports@googlegroups.com here ?
As per  https://istio.io/latest/about/security-vulnerabilities/

Reply:
That will not be a problem. For the record though, this will probably not work with getting access to bug reports. I have seen examples of this being the case.
It will work fine for notifications though.